resources/js/pages/home/Components/Story/StoryDetails.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import './story.css';
+import Story from '../../../../models/Story';
+import StoryModal from './StoryModal';
+import StoryItem from './StoryItem';
+
+
+export const StoryDetails = () => {
+    // Getting the story data from the Story model
+    const [userStories, setUserStories] = React.useState<Story[]>([]);
+    const [selectedStory, setSelectedStory] = React.useState<Story | null>(null);
+
+    React.useEffect(() => {
+        const fetchStories = async () => {
+            try {
+                const storyList = await Story.all();
+                const formattedStories = storyList.map((story) => ({
+                    url: story.image,
```

Comment:
you are getting a error related to props check console its not accepting props you have entered